guides/source/active_record_migrations.md

```diff
@@ -220,88 +240,83 @@ class RemovePartNumberFromProducts < ActiveRecord::Migration[7.2]
 end
 ```
 
-### Creating New Tables
+### Creating Associations
```

Comment:
This is the first mention in this guide about Associations. Should we link somehow to association_basics.html?